roles/postgres_replicant/tasks/main.yml

```diff
@@ -47,6 +48,7 @@
     mode: 0640
 
 - name: start postgresql service
+  become: yes
   service:
     name: postgresql
```

Comment:
The change I made when we did the deploy was to change the service name to postgresql@9.4-main.service